src/utils/cassandra.js

```diff
@@ -66,22 +66,42 @@ export function getCassandraClientAsync(keyspace, dseUsername, dsePassword) {
         clientOpts.keyspace = keyspace;
       }
 
-       /** 
-       * Check for both KILLRVIDEO_DSE_USERNAME and KILLRVIDEO_DSE_PASSWORD environment
-       * variables.  If they both exist use the values set within them.  If not,
-       * use default values for authentication.
-       */
+      /**
+      * Check for both KILLRVIDEO_DSE_USERNAME and KILLRVIDEO_DSE_PASSWORD environment
+      * variables.  If they both exist use the values set within them.  If not,
+      * use default values for authentication.
+      */
       if (dseUsername && dsePassword) {
         let passwordLength = dsePassword.length;
-        logger.info('Using supplied DSE username: "' + dseUsername + '" and password: "***' + dsePassword.substring(passwordLength - 4, passwordLength) + '" from environment variables')
+        logger.info('Using supplied DSE username: "' + dseUsername + '" and password: "***' + dsePassword.substring(passwordLength - 4, passwordLength) + '" from environment variables');
 
         // Use the values passed in from the config
         clientOpts.authProvider = new auth.DsePlainTextAuthProvider(dseUsername, dsePassword);
 
       } else {
         logger.info('No detected username/password combination was passed in. DSE cluster authentication method was NOT executed.');
       }
-      
+
+      let Filesystem = require("fs");
```

Comment:
See my question on the Java implementation about whether we should continue to execute if SSL enabled but there is a problem with the cert file.

Reply:
@SonicDMG and I discussed this a bit, and if I recall agreed to go ahead and get this in as is, and work on figuring out how to determine if the cert file is ok via code at a later time. I wasn't sure exactly how to go about doing that at this point.